dashboard/src/components/AppView/AppView.tsx

```diff
@@ -31,11 +32,10 @@ import {
   IStoreState,
   NotFoundNetworkError,
 } from "shared/types";
+import * as url from "shared/url";
 import { getPluginsSupportingRollback } from "shared/utils";
-import { parseToString, parseToJS } from "shared/yamlUtils";
+import { parseToJS, parseToString } from "shared/yamlUtils";
```

Comment:
Feel free to add an alphabetic sorting to our prettier or lint so you don't need to clean them up :)

Reply:
Yeah, it would be a great addition. I've seen https://eslint.org/docs/latest/rules/sort-imports and https://github.com/trivago/prettier-plugin-sort-imports. Will have a look later to see if they work for us.